common/src/service/HttpClientHelper.cs

```diff
@@ -34,21 +34,24 @@ internal sealed class HttpClientHelper : IHttpClientHelper
         private readonly TimeSpan _defaultOperationTimeout;
         private readonly IWebProxy _customHttpProxy;
         private readonly Action<HttpClient> _preRequestActionForAllRequests;
+        private readonly bool _certificateRevocationCheck;
 
         public HttpClientHelper(
             Uri baseAddress,
             IAuthorizationHeaderProvider authenticationHeaderProvider,
             IDictionary<HttpStatusCode, Func<HttpResponseMessage, Task<Exception>>> defaultErrorMapping,
             TimeSpan timeout,
             Action<HttpClient> preRequestActionForAllRequests,
-            IWebProxy customHttpProxy)
+            IWebProxy customHttpProxy,
+            bool certificateRevocationCheck)
         {
             _baseAddress = baseAddress;
             _authenticationHeaderProvider = authenticationHeaderProvider;
             _defaultErrorMapping = new ReadOnlyDictionary<HttpStatusCode, Func<HttpResponseMessage, Task<Exception>>>(defaultErrorMapping);
             _defaultOperationTimeout = timeout;
             _preRequestActionForAllRequests = preRequestActionForAllRequests;
             _customHttpProxy = customHttpProxy;
+            _certificateRevocationCheck = certificateRevocationCheck;
```

Comment:
I might be missing something here, but where is _certificateRevocationCheck  every used in this file?

Reply:
we are basically setting property this for httpClientHelper constructor which is referenced in other places such as HttpRegistryManager.cs